app/controllers/likes_controller.rb

```diff
@@ -2,7 +2,11 @@ class LikesController < ApplicationController
   def create
     @post = Post.find(params[:post_id])
     @like = @post.likes.new(user: current_user)
-    @like.update_counter if @like.save
-    redirect_to [@post.user, @post]
+    if @like.save
+      @like.update_counter
+      flash[:notice] = 'You successfully liked the post.'
+    else
+      flash[:error] = 'You were unable to give this post a like.'
```

Comment:
- [ ] For all the `posts#create`, `comments#create`, and `likes#create` actions you have used flash messages for a successful and not successful path but it is not rendering in the view. Please make sure to show these flash message in the view as well. :smile: